common/lib/xmodule/xmodule/tests/test_lti_unit.py

```diff
@@ -171,9 +171,9 @@ def test_real_user_is_none(self):
         """
         If we have no real user, we should send back failure response.
         """
+        self.system._services['user'] = StubUserService(user=None)  # pylint: disable=protected-access
```

Comment:
Why can't we use `self.system.service(self.xmodule, 'user').user = None` here, like in the test from `common/lib/xmodule/xmodule/tests/test_lti20_unit.py`?

Reply:
Actually, they should both be using the `StubUserService` mechanism instead, because the real `DjangoXBlockUserService` class doesn't have a `user` attribute. see https://github.com/edx/edx-platform/pull/29375#discussion_r757109450 for discussion.

Good catch, I missed this one!